internal/api/v1beta1connect/session.go

```diff
@@ -0,0 +1,186 @@
+package v1beta1connect
+
+import (
+	"context"
+	"errors"
+	"strings"
+	"time"
+
+	"connectrpc.com/connect"
+	"github.com/google/uuid"
+	"github.com/raystack/frontier/core/authenticate"
+	frontiersession "github.com/raystack/frontier/core/authenticate/session"
+	frontiererrors "github.com/raystack/frontier/pkg/errors"
+	sessionutils "github.com/raystack/frontier/pkg/session"
+	frontierv1beta1 "github.com/raystack/frontier/proto/v1beta1"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// SDK APIs
+// Returns a list of all sessions for the current authenticated user.
+func (h ConnectHandler) ListSessions(ctx context.Context, request *connect.Request[frontierv1beta1.ListSessionsRequest]) (*connect.Response[frontierv1beta1.ListSessionsResponse], error) {
```

Comment:
```suggestion
func (h *ConnectHandler) ListSessions(ctx context.Context, request *connect.Request[frontierv1beta1.ListSessionsRequest]) (*connect.Response[frontierv1beta1.ListSessionsResponse], error) {
```

Struct ConnectHandler has methods on both value and pointer receivers. Such usage is not recommended by the Go Documentation.